src/sentry/testutils/factories.py

```diff
@@ -95,16 +95,11 @@
 from sentry.utils import json, loremipsum
 
 
-def get_fixture_path(name):
-    return os.path.join(
-        os.path.dirname(__file__),  # src/sentry/testutils/
-        os.pardir,  # src/sentry/
-        os.pardir,  # src/
-        os.pardir,
-        "tests",
-        "fixtures",
-        name,
-    )
+def get_fixture_path(*parts: str) -> str:
```

Comment:
What wasn't correct or optimal about this previously? Just curious.

Reply:
it left a bunch of `..`s in the path, the path wasn't absolute so it could potentially cut off the root early, and it had the old path